using Microsoft.Extensions.Configuration;

namespace Shared.Exceptions.Messages
{
    public class ExceptionMessages
    {
        public const string USER_DOESNOT_EXIST = "User does not exist.";
        public const string USER_ALREADY_EXIST = "User already exists.";
        public const string INVALID_CREDENTIALS = "Invalid credentials.";
        public const string DOMAIN_ALREADY_EXIST = "Domain already exists.";
        public const string DOMAIN_CONFIGURATION_ISSUE = "Domain configuration issue, please retry another domain name.";
        public const string APP_DOESNOT_EXIST = "App does not exist.";
        public const string RECORD_NOT_FOUND = "Record not found";
        public const string FAILED = "Failed";
        public const string Email_ALREADY_EXIST = "Email already exists.";
        public const string SUBSCRIPTION_DOESNOT_EXIST = "Subscription does not exist.";
        public const string SUBSCRIPTION_ALREADY_EXIST = "Subscription already exists.";


        public const string BLOG_DOESNOT_EXIST = "Blog does not exist.";

    }
}
